src/clib/initialize_cloudy_data.c

```diff
@@ -24,6 +24,21 @@
 
 extern int grackle_verbose;
 
+
+/**
+ * Initializes an empty #cloudy_data struct with zeros and NULLs.
+ */
+void initialize_empty_cloudy_data_struct(cloudy_data *my_cloudy)
```

Comment:
I note this is never called. It looks like it should be for both `cloudy_primordial` and `cloudy_metal`.

Reply:
mea culpa. Fixed.